# C_plus_try_catch_throw
C++中try/catch/throw的使用
